### ISSUE TITLE
Web Raster is Rendering Containers as divs

### ISSUE THREAD

**Issue by dmikey** [dmikey/syr#41]
Outer component is rendering a DIV as base tag. This should be changed. I think I picked this incase someone didn't add a Tag, but now I'm seeing this as a downside :) to our Functional Classes thing. lets remove this.

**Comment by dmikey** [comment]
![screen shot 2017-12-18 at 10 56 37 am](https://user-images.githubusercontent.com/328000/34122732-3f3eb07c-e3e2-11e7-90d9-baccc1d1c777.png)

```
class Address extends Component {
  render(){
    return <View style={{
        left: 24,
        width: Dimensions.get('window').width - 64,
        height: 80,
        top: 75
      }}>
```

**Comment by MSiddharthReddy** [comment]
Resolved in Jsx Changes #45.
